client/webserver/locales/en-us.go

```diff
@@ -217,7 +218,8 @@ var EnUS = map[string]string{
 	"Maximum Possible Swap Fees":  "Maximum Possible Swap Fees",
 	"max_fee_conditions":          "This is the most you would ever pay in fees on your swap. Fees are normally assessed at a fraction of this rate. The maximum is not subject to changes once your order is placed.",
 	"wallet_logs":                 "Wallet Logs",
-	"accelerate_order":            "Accelerate Order",
+	":title:accelerate_order":     "Accelerate Order",
+	"accelerate_order":            "accelerate order",
```

Comment:
The second suggestion in https://github.com/decred/dcrdex/pull/1902#discussion_r996459520 is to avoid needing two definitions at all, only needing the non-title one.

In the locales files we'd only have `"accelerate_order":            "accelerate order",` (for each lang), the tmpl would use `[[[:title:accelerate_order]]]` when title case is desired, and then use `golang.org/x/text/cases.Caser` for the title case version.

```diff
diff --git a/client/webserver/template.go b/client/webserver/template.go
index 554ecf3c..ca98aacb 100644
--- a/client/webserver/template.go
+++ b/client/webserver/template.go
@@ -12,6 +12,7 @@ import (
        "strings"
 
        "decred.org/dcrdex/client/webserver/locales"
+       "golang.org/x/text/cases"
 )
 
 // pageTemplate holds the information necessary to process a template. Also
@@ -27,6 +28,7 @@ type templates struct {
        fs           fs.FS // must contain tmpl files at root
        reloadOnExec bool
        dict         map[string]string
+       titler       cases.Caser
 
        addErr error
 }
@@ -70,6 +72,11 @@ func (t *templates) translate(name string) (string, error) {
                        return "", fmt.Errorf("can't parse match group: %v", matchGroup)
                }
                token, key := matchGroup[0], string(matchGroup[1])
+               var toTitle bool
+               if titleKey := strings.TrimPrefix(key, ":title:"); titleKey != key {
+                       toTitle = true
+                       key = titleKey
+               }
                replacement, found := t.dict[key]
                if !found {
                        replacement, found = locales.EnUS[key]
@@ -77,6 +84,9 @@ func (t *templates) translate(name string) (string, error) {
                                return "", fmt.Errorf("warning: no translation text for key %q", key)
                        }
                }
+               if toTitle {
+                       replacement = t.titler.String(replacement)
+               }
                rawTmpl = bytes.ReplaceAll(rawTmpl, token, []byte(replacement))
```

Reply:
In another work, I will update duplicated translations that could use the `:title:` prefix.